cvat/requirements/base.txt

```diff
@@ -50,4 +50,4 @@ urllib3>=1.26.5 # not directly required, pinned by Snyk to avoid a vulnerability
 natsort==8.0.0
 mistune>=2.0.1 # not directly required, pinned by Snyk to avoid a vulnerability
 dnspython==2.2.0
-setuptools>=65.5.1 # not directly required, pinned by Snyk to avoid a vulnerability
+setuptools==65.5.1
```

Comment:
Should probably link https://github.com/numpy/numpy/issues/22623 for explanation.